Beeper/barcelona-mautrix/SendMessageCLICommand.swift

```diff
@@ -37,15 +39,48 @@ class SendMessageCLICommand: Command {
             exit(1)
         }
 
+        if let overwriteAvailability {
+            IDSResolver.overwrittenStatuses[chat.chatIdentifier] = overwriteAvailability ? 1 : 2
+        }
+
         let messageCreation = CreateMessage(parts: [
             .init(type: .text, details: self.message ?? "Test Message")
         ])
 
         log.info("Message send task starting")
-        let _ = try! await chat.send(message: messageCreation)
+        let msg = try! await chat.send(message: messageCreation)
 
-        log.info("Message sent!")
-        exit(0)
+        log.info("Message sent: \(msg)\nAwaiting statuses...")
+
+        CBDaemonListener.shared.messageStatusPipeline.retainingSink { comp in
```

Comment:
Nit: I *think* you can avoid the retaining sink by just storing the `Cancellable` in a property on the command, it should align with the lifetime nicely. But I might be wrong.